wfe/web-front-end.go

```diff
@@ -152,6 +155,9 @@ func (wfe *WebFrontEndImpl) NewAuthorization(response http.ResponseWriter, reque
 		return
 	}
 
+  // XXX The spec says a client should send an Accept: application/pkix-cert
```

Comment:
Personally I prefer TODOs in checked-in code and XXX as a signal to myself that something should be fixed before checking it.